src/test/scala/pt/tecnico/dsi/openstack/designate/QuotasSpec.scala

```diff
@@ -0,0 +1,66 @@
+package pt.tecnico.dsi.openstack.designate
+
+import cats.effect.IO
+import org.http4s.Header
+import pt.tecnico.dsi.openstack.designate.models.Quota
+import pt.tecnico.dsi.openstack.keystone.models.Project
+
+class QuotasSpec extends Utils {
+  val allProjectsHeader: Header = Header("x-auth-all-projects", "true")
+  val withStubProject: IO[(DesignateClient[IO], String)] =
+    for {
+      keystone <- keystoneClient
+      dummyProject <- keystone.projects.create(Project("dummy", "dummy project", "default"))
+      designate <- client
+    } yield (designate, dummyProject.id)
+
+  "The Quotas service" should {
+    "get quotas for the current project" in withStubProject.flatMap { case (designate, _) =>
+      designate.quotas.get().idempotently { quotas =>
+        quotas.apiExportSize shouldBe 20
+        quotas.recordsetRecords shouldBe 30
+        quotas.zoneRecords shouldBe 10
+        quotas.zoneRecordsets shouldBe 25
+        quotas.zones shouldBe 15
+      }
+    }
+
+    "get quotas" in withStubProject.flatMap { case (designate, dummyProjectId) =>
+      designate.quotas.get(dummyProjectId, allProjectsHeader).idempotently { quotas =>
+        quotas.apiExportSize shouldBe 1000
+        quotas.recordsetRecords shouldBe 20
+        quotas.zoneRecords shouldBe 500
+        quotas.zoneRecordsets shouldBe 500
+        quotas.zones shouldBe 10
+      }
+    }
+
+    "update quotas" in withStubProject.flatMap { case (designate, dummyProjectId) =>
+      val newQuotas = Quota(
+        apiExportSize = 20,
+        recordsetRecords = 30,
+        zoneRecords = 10,
+        zoneRecordsets = 25,
+        zones = 15
+      )
+      designate.quotas.update(dummyProjectId, newQuotas, allProjectsHeader).idempotently( _ shouldBe newQuotas)
```

Comment:
This makes a lot of sense. But the previous test, where we check for _our_ defaults doesn't seem right. Afterall, the test will not pass if the defaults have changed.

Reply:
You have no way of knowing the default quotas without asking for them. And if you did that the test would be useless.

Since this is a client we are not really interested in the real value of the quotas/default quotas but rather that the client can successfully parse them.

Reply:
What about an update followed by a get, instead of just a get? This way the test would always pass inrespective of the defaults, since you would be updating it first. At the end of the day, the integration test would continue to test update and get, which is what we want.